cleanup/mavenSnapshotCleanupWhenRelease/README.md

```diff
@@ -0,0 +1,27 @@
+Artifactory Maven Snapshot Cleanup When Release
+=====================================
+
+This plugin deletes Maven SNAPSHOT artifacts when the release version is published.
+
+
+Sample
+----------
+
+For this sample, *mavenSnapshotCleanupWhenRelease.properties* file contains:
+
+    repositories =  [ ["maven-local-lib-releases","maven-local-lib-snapshots"] ]
+
+Consider the Maven snapshots repository `maven-local-lib-snapshots` populated like that:
+
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-1.pom
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-1.jar
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-2.pom
+    org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/my-artifact-1.0.0-20170101.080829-2.jar
+
+    org/jfrog/test/my-artifact/1.1.0-SNAPSHOT/my-artifact-1.1.0-20170202.080829-1.pom
+    org/jfrog/test/my-artifact/1.1.0-SNAPSHOT/my-artifact-1.1.0-20170202.080829-1.jar
+
+
+When the **pom file** of `my-artifact` version `1.0.0` is deployed into `maven-local-lib-releases` (Maven releases repository), the associated snapshot is deleted.
+
+e.g: folder `org/jfrog/test/my-artifact/1.0.0-SNAPSHOT/` of `maven-local-lib-snapshots`.
```

Comment:
@axel3rd the README file must be at least the following sections:
- **Description**: A brief description about the plugin.
- **Features**: More detailed description about the features. You can put the sample here.
- **Installation**: How to install and setup it.
- **Execution**: How to use it.